src/schema/v2/viewingRooms/mutations/createViewingRoomMutation.ts

```diff
@@ -36,37 +17,7 @@ export const createViewingRoomMutation = mutationWithClientMutationId<
     // This is because Gravity has such duplication https://github.com/artsy/gravity/blob/main/app/graphql/mutations/create_viewing_room.rb#L12
     // We can get rid of it once we finish with the migration. For now I want to keep such changes to a minimum
     attributes: {
-      type: new GraphQLInputObjectType({
-        name: "ViewingRoomAttributes",
-        fields: {
-          body: {
-            type: GraphQLString,
-          },
-          endAt: {
-            type: GraphQLString,
-            description: "Datetime (in UTC) when Viewing Room closes",
-          },
-          introStatement: {
-            type: GraphQLString,
-          },
-          pullQuote: {
-            type: GraphQLString,
-          },
-          startAt: {
-            type: GraphQLString,
-            description: "Datetime (in UTC) when Viewing Room opens",
-          },
-          timeZone: {
-            type: GraphQLString,
-            description:
-              "Time zone (tz database format, e.g. America/New_York) in which start_at/end_at attributes were input",
-          },
-          title: {
-            type: GraphQLString,
-            description: "Title",
-          },
-        },
-      }),
+      type: ViewingRoomInputAttributesType,
```

Comment:
**question (non-blocking):** Confirming that the comments above are still relevant?

Reply:
Yep, still accurate - I just extracted this code block to another file to re-use it elsewhere @anandaroop